src/components/UI/ResourcesLink/ResourcesLink.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "@emotion/styled";
+import React, { FunctionComponent } from "react";
+import { Download } from "react-feather";
+import { mixin, vars } from "../../../styles";
+
+interface ResourcesLinkProps {
+  className?: string;
+  title: string;
+  details: {
+    description: string;
+    url: string;
+    icon?: string;
+  };
+  type: string;
+}
```

Comment:
type only has `link` or `download` right? maybe can put it `"link" | "download"` so we'll know only got these 2 options to use

Reply:
ok